src/Microsoft.Data.SqlClient/add-ons/AzureKeyVaultProvider/AKVEventSource.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+using System.Threading;
+
+namespace Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider
+{
+    [EventSource(Name = "Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider.EventSource")]
+    internal class AKVEventSource : EventSource
+    {
+        // Defines the singleton instance for the Resources ETW provider
+        public static AKVEventSource Log = new();
```

Comment:
It's a good practice to hide the default ctor in a singleton approach by explicitly defining a private ctor. It seems we missed this part on SqlClientEventSource too.